src/hammer-vlsi/synthesis/vivado/defaults.yml

```diff
@@ -0,0 +1,22 @@
+# Default settings for synthesis in Vivado, for project/technology configuration and overriding.
+synthesis.vivado:
+  # Location of the binary.
+  binary: "vivado"
+  # Location of the vivado setup script
+  setup_script: "/tools/projects/erichang/hammer-examples/vivado_setup.sh"
+  # Location of the constraints file
+  constraints_file: "/tools/projects/erichang/hammer-examples/vc707.xdc"
```

Comment:
Ditto above

Reply:
Yes I agree in general they need not (and probably aren't) in the tool directory. It would be nice to have it work "out of the box" in some way and hardcoding BWRC-specific paths in here might not be the best idea.

I'd envision that these could point to the default scripts and generators that use Hammer can customize/override them as they wish.